lib/roo_on_rails/sidekiq/queue_latency.rb

```diff
@@ -5,6 +5,29 @@ module Sidekiq
     class QueueLatency
       extend Forwardable
 
+      def self.permitted_latency_values
+        @permitted_latency_values ||= Hash.new do |hash, queue_name|
+          hash[queue_name] = begin
+            case queue_name
+            when 'monitoring', 'realtime'
+              10.seconds.to_i
+            when 'default'
+              1.day.to_i
+            when /\Awithin\d+.+\z/
+              _, number, unit = queue_name.partition(/\d+/)
+              number.strip.to_i.public_send(unit.strip).to_i
+            else
+              ENV.fetch('SIDEKIQ_QUEUES', '').split(',').reduce(nil) do |result, entry|
```

Comment:
This loop here I found hard to understand. Maybe it would be simpler to build a `{name => latency}` hash of the `SIDEKIQ_QUEUES` variable once at startup and then this `else` block would just be a hash lookup?